src/swdmt/redistricting/LocationTest.java

```diff
@@ -34,13 +33,13 @@ public void coordinateValuesTest() {
     @Test
     public void equalsSelfTest() {
         Location location1 = new Location(17, 23);
-        assertTrue(location1.equals(location1));
+        assertEquals(location1, location1);
     }
 
     @Test
     public void equalsNullTest() {
         Location location1 = new Location(7, 13);
-        assertFalse(location1.equals(null));
+        assertNotEquals(null, location1);
```

Comment:
Is this part of the JUnit4 to JUnit5 conversion? Is assertTrue not in JUnit5?
Rewriting tests for other reasons is not part of the issue.

Reply:
assertFalse is part of JUnit5. I reverted the change.